nengo_bones/templates/remote.sh.template

```diff
@@ -93,12 +93,36 @@ See https://docs.travis-ci.com/user/encrypting-files/ for more details.
         REMOTE_STATUS=0
         cd ~/"$BUILD_DIR"
         {% block remote_install %}
-        echo "$ ({{ host }}) Installing miniconda"
-        wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh --quiet -O miniconda.sh
-        bash miniconda.sh -b -p ./miniconda
+        echo "$ ({{ host }}) Downloading miniconda"
+        DOWNLOAD_STATUS=0
+        wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh --quiet -O miniconda.sh || DOWNLOAD_STATUS=1
+        if [ "\$DOWNLOAD_STATUS" -gt 0 ]; then
+            echo "({{ host }}) Failed to download miniconda. Exiting.";
+            exit \$DOWNLOAD_STATUS;
+        fi
+        for i in \$(seq 1 10); do
```

Comment:
I don't know if the potential benefit of an installation failing the first time but succeeding on a later attempt is worth all the added complexity here. It's already rare that the installation fails, and it'd be even more rare that it would succeed on a later attempt when that does occur. I'd be inclined to just try the installation once, and exit on failure. On the rare occasions that that happens, people can always just restart the build to try again.

Reply:
It's become much more common that the installation fails (see many of the last builds on TravisCI.org, as well as at least one of the new ones on TravisCI.com). I believe it may be because of a flaky internet connection where our machine is hosted. Restarting automatically saves considerable time and hassle.

Here's one example of the restarting working successfully: https://travis-ci.com/github/nengo/nengo-bones/jobs/449899508

Reply:
It's gotta be an `abrgl` issue, because I've literally never seen it fail in NengoDL (which is running on Azure instead). But yeah if it's happening enough to be noticeable on `abrgl` then I guess this is worth it.